_assets/compose/mailserver/Makefile

```diff
@@ -14,6 +14,8 @@ export API_MODULES    ?= eth,net,web3,admin
 export CONTAINER_NAME ?= status-go-mailserver
 export DATA_PATH      ?= /var/tmp/status-go-mail
 export FLEET_NAME     ?= eth.beta
+export MAIL_TOPIC     ?= whispermail
```

Comment:
FYI this topic is for dynamic mail server discovery using our peers discovery mechanisms.

Reply:
Ok, renamed variable to `REGISTER_TOPIC` and updated `README`.